src/Ships/P1MissileCorvette.c

```diff
@@ -174,22 +182,42 @@ bool P1MissileCorvetteSpecialTarget(Ship *ship,void *custom)
     {
         // have destroyed targets, so we are done
         spec->curTargetIndex = 0;
+        spec->volleyState = VOLLEY_BEGIN;
         return TRUE;
     }
 
+    if(spec->volleyState == VOLLEY_BEGIN)
+    {
+        spec->volleyState = VOLLEY_NOT_BEGIN;
+        //////////////////////
+        //Volley attack speech event!
+        //event num: COMM_MissleDest_VolleyAttack
+        //use battle chatter
+        if(ship->playerowner->playerIndex == universe.curPlayerIndex)
+        {
+            if (battleCanChatterAtThisTime(BCE_COMM_MissleDest_VolleyAttack, ship))
+            {
+                battleChatterAttempt(SOUND_EVENT_DEFAULT, BCE_COMM_MissleDest_VolleyAttack, ship, SOUND_EVENT_DEFAULT);
+            }
+        }
+        //////////////////////
+    }
+
```

Comment:
[M] My guess is that with almost all special ability behaviors shared, everything here can be safely reused.